src/structs.rs

```diff
@@ -112,6 +112,41 @@ pub struct IssueAssetRequest {
     pub request: IssueRequest,
 }
 
+#[derive(Serialize, Deserialize, Debug, Clone, Default)]
+#[serde(rename_all = "camelCase")]
+#[derive(Validate)]
+#[garde(context(RGBContext))]
+pub struct IssuePreRequest {
```

Comment:
Just for naming convention, similar to FullTransferRequest.

```suggestion
pub struct FullIssueRequest {
```